### PR TITLE
docs: update the package table for my pepz :)

### PR DESCRIPTION
**Pull request by daviddias**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -563,12 +563,10 @@ List of packages currently in existence for libp2p
...
+| [`libp2p-daemon`](//github.com/libp2p/js-libp2p-daemon) | [![npm](https://img.shields.io/npm/v/libp2p-daemon.svg?maxAge=86400&style=flat-square)](//github.com/libp2p/js-libp2p-daemon/releases) | [![Deps](https://david-dm.org/libp2p/js-libp2p-daemon.svg?style=flat-square)](https://david-dm.org/libp2p/js-libp2p-daemon) | [![Travis CI](https://travis-ci.com/libp2p/js-libp2p-daemon.svg?branch=master)](https://travis-ci.com/libp2p/js-libp2p-daemon) | [![codecov](https://codecov.io/gh/libp2p/js-libp2p-daemon/branch/master/graph/badge.svg)](https://codecov.io/gh/libp2p/js-libp2p-daemon) | [Jacob Heun](mailto:jacobheun@gmail.com) |
```
Comment (vasco-santos) [inline README.md]: Can you also add the [libp2p/js-libp2p-daemon-client](https://github.com/libp2p/js-libp2p-daemon-client) here?

Comment (daviddias) [reply README.md]: ah, of course! thanks for the reminder

File: README.md [hunk 2]
```diff
@@ -578,34 +576,35 @@ List of packages currently in existence for libp2p
...
-| **Extensions** |
+| [`peer-book`](//github.com/libp2p/js-peer-book) | [![npm](https://img.shields.io/npm/v/peer-book.svg?maxAge=86400&style=flat-square)](//github.com/libp2p/js-peer-book/releases) | [![Deps](https://david-dm.org/libp2p/js-peer-book.svg?style=flat-square)](https://david-dm.org/libp2p/js-peer-book) | [![Travis CI](https://travis-ci.com/libp2p/js-peer-book.svg?branch=master)](https://travis-ci.com/libp2p/js-peer-book) | [![codecov](https://codecov.io/gh/libp2p/js-peer-book/branch/master/graph/badge.svg)](https://codecov.io/gh/libp2p/js-peer-book) | [Vasco Santos](mailto:vasco.santos@moxy.studio) |
```
Comment (vasco-santos) [inline README.md]: We are going to deprecate the `peer-book`. We already have the new `peer-store` withing this repo, so I think we can remove this now

Comment (vasco-santos) [reply README.md]: I mean, it is on https://github.com/libp2p/js-libp2p/tree/refactor/async-await but we intend to merge it today/tomorrow

Comment (daviddias) [reply README.md]: Once it is done. You can then update the list. Takes a minute really :)

**Comment by jacobheun** [comment]
This removed a lot of the lead maintainers from the table for some reason. Ideally I would have liked to have moved this into the async branch as that is going to be released soon and we'll need to resolve the conflicts with this now.

We can fix the lead maintainer issues in the table when we rebase into async. 

@daviddias in the future can you make sure the PR is approved before merging? 🙏

**Comment by daviddias** [comment]
@jacobheun ah you are right. My apologies, for some reason, my brain parsed it as approved with just a couple of nits to fix. My apologies!

Yeah, now that I open each package, I see that there is indeed a leadMaintainer that didn't get populated. (failed silently)

**Pull request merged**